monitoring/README.md

```diff
@@ -194,7 +197,14 @@ You'll need to run a few more steps to completely setup the AAD Service Principa
 az keyvault set-policy --secret-permissions set --object-id $(az ad signed-in-user show --query id -o tsv) -n $ASB_KV_NAME -g $ASB_RG_CORE
 
 # Set grafana AAD client secrets
-az keyvault secret set -o table --vault-name $ASB_KV_NAME --name "grafana-aad-client-secret" --value "$CLIENT_SECRET"
+az keyvault secret set -o table --vault-name $ASB_KV_NAME --name "grafana-aad-client-secret" --value $GRAFANA_OAUTH_CLIENT_SECRET
+
+az keyvault secret set -o table --vault-name $ASB_KV_NAME --name "grafana-azure-monitor-client-secret" --value $AZ_MONITOR_CLIENT_SECRET
+
+# set grafana AAD ids
+export ASB_GRAFANA_SP_CLIENT_ID=[insert CLIENT_ID]
+export ASB_GRAFANA_SP_TENANT_ID=[insert TENANT_ID]
```

Comment:
Would this be the same values set above in line 153 and 155 for `CLIENT_ID` and `TENANT_ID`?

```suggestion
export ASB_GRAFANA_SP_CLIENT_ID=$CLIENT_ID
export ASB_GRAFANA_SP_TENANT_ID=$TENANT_ID
```